services/decoder/protocols/wormhole/wormhole.decoders.ts

```diff
@@ -0,0 +1,126 @@
+import { GoldRushDecoder } from "../../decoder";
+import type { EventDetails } from "../../decoder.types";
+import { type EventType } from "../../decoder.types";
+import {
+    DECODED_ACTION,
+    DECODED_EVENT_CATEGORY,
+} from "../../decoder.constants";
+import { decodeEventLog, type Abi } from "viem";
+import PORTAL_BRIDGE_ABI from "./abis/wormhole-portal-bridge.abi.json";
+import ETH_CORE_ABI from "./abis/wormhole-eth-core.abi.json";
+
+GoldRushDecoder.on(
+    "wormhole:TransferRedeemed",
+    ["eth-mainnet"],
```

Comment:
You have added configs for many chains, but decoding only on `eth-mainnet`
Either the other configs are not needed or the remaining chains need to be added here

Reply:
Adding.

Reply:
https://github.com/covalenthq/goldrush-decoder/pull/46/commits/61092e564f8907c68114b5bda40e158395134a16#diff-9cd52e11d7319bb7cfbf40c8abd49e83bfb4cd2ab23846e3807a7015b35c9157R14